src/js/components/Cards/stories/Children.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Card, CardBody, CardFooter, Grid, Heading } from 'grommet';
+import { Cards } from '../Cards';
+
+const data = [
+  { city: 'Boise', state: 'Idaho' },
+  { city: 'Fort Collins', state: 'Colorado' },
+  { city: 'Bay Area', state: 'California' },
+  { city: 'San Diego', state: 'California' },
+  { city: 'San Francisco', state: 'California' },
+];
+
+export const Children = () => (
+  <Grid pad="large" columns={[['medium', 'large']]} justifyContent="center">
```

Comment:
Accessibility error on this story: `<ul> and <ol> must only directly contain <li>, <script> or <template> elements`

Reply:
Fixed.